nixos/modules/security/pam.nix

```diff
@@ -648,6 +650,51 @@ let
          "${domain} ${type} ${item} ${toString value}\n")
          limits);
 
+  limitsType = with lib.types; listOf (submodule ({ ... }: {
+    options = {
+      domain = mkOption {
+        description = "Username, groupname, or wildcard this limit applies to";
+        example = "@wheel";
+        type = str;
+      };
+
+      type = mkOption {
+        description = "Type of this limit";
+        type = enum [ "-" "hard" "soft" ];
```

Comment:
What's `-` here? Might need another comment - or maybe some links to docs.

Reply:
I linked to the docs in the main option